site/get-opa/index.md

```diff
@@ -0,0 +1,51 @@
+---
+nav_id: MAIN_GET_OPA
+layout: default
+
+title: Get Open Policy Agent
+---
+
+{% contentfor header %}
+
+# Get Open Policy Agent
+{: .opa-header--minor-title}
+
+OPA is available as binary releases for 64-bit Linux and Mac OS X or as Golang source code.
+{: .opa-header--text}
+
+  * [64-bit Linux](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa_linux_amd64){: .opa-header--download-list--link}
+  * [64-bit Mac OS X](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa_darwin_amd64){: .opa-header--download-list--link}
+  * [Go Source](https://github.com/open-policy-agent/opa/releases/download/v0.1.0-rc3/opa0.1rc3.src.tar.gz){: .opa-header--download-list--link}
```

Comment:
I don't think you can do that with releases. I looked at a few other projects that distribute binaries through GitHub releases and this seems to be the model.